frontend/Data.js

```diff
@@ -118,4 +118,46 @@ export default class Data {
       return e.response.data.errors
     }
   }
+  async deleteComment(postID, commentID) {
+    try {
+      const response = await this.api(
+        'post/' + postID + '/' + commentID,
+        'DELETE',
+        null
+      )
+      if (response.status === 204) {
+        return response
+      } else if (response.status === 400) {
+        return response.json().then(data => {
+          return data.errors
+        })
+      } else {
+        throw new Error(response.message)
+      }
+    } catch (e) {
+      console.log(e)
+      return e.response.data.errors
```

Comment:
this could cause its own issues because the error object isn't guaranteed to have a `data` field